data/index.json

```diff
@@ -76,18 +76,126 @@
         },
         "registry": [
             { "group": "other", "id": "alzheimer", "name": "Alzheimer" },
```

Comment:
Same here. This is probably a more serious issue, since e.g. `ischemicheartdisease` cannot be considered in the simulation.